backendservice/src/main/java/com/google/gpay/virtualqueue/backendservice/service/ShopService.java

```diff
@@ -0,0 +1,18 @@
+package com.google.gpay.virtualqueue.backendservice.service;
+
+import com.google.gpay.virtualqueue.backendservice.entities.Shop;
+import com.google.gpay.virtualqueue.backendservice.respository.ShopRepository;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+@Service
+public class ShopService {
+
+	@Autowired
+	ShopRepository shopRepository;
+
+	public String CreateShop(Shop shop) {
```

Comment:
Nit. Method names should begin with small alphabet. Here and in other places.

Reply:
Done!